docs/api-reference/index.md

```diff
@@ -9,6 +9,11 @@
    :toctree: ../generated/classes
    :template: class-template.rst
    :recursive:
+
+   SciTiff
+   SciTiffMetadataContainer
+   SciTiffMetadata
```

Comment:
pydantic class does not render as expected. There is a documentation plugin for pydantic model so we'll use that one...  #4